docs/content/concepts/ops-jobs-graphs/ops.mdx

```diff
@@ -162,35 +162,40 @@ Like inputs, outputs can also have [Dagster Types](/concepts/types).
 
 While many use cases can be served using built-in python annotations, <PyObject object="Output"/> and <PyObject object="DynamicOutput"/> objects unlock additional functionality. Check out the docs on [Op Outputs](/concepts/ops-jobs-graphs/op-events#output-objects) to learn more.
 
-### Op Context
+### Op Configuration
 
-When writing an op, users can optionally provide a first parameter, `context`. When this parameter is supplied, Dagster will supply a context object to the body of the op. The context provides access to system information like op configuration, loggers, resources, and the current run id. See <PyObject object="OpExecutionContext"/> for the full list of properties accessible from the op context.
+Ops in Dagster can specify a config schema which makes them configurable and parameterizable. The configuration system is explained in detail in the [Config schema documentation](/concepts/configuration/config-schema).
 
-For example, to access the logger and log a info message:
+Op functions can specify an annotated `config` parameter for the op's configuration. The config class, which subclasses <PyObject object="Config"/>, specifies the configuration schema for the op. Op configuration can be used to specify op behavior at runtime, making ops more flexible and reusable.
```

Comment:
I think we should emphasize in the docs that these in turn inherit from `pydantic.BaseModel` and be very transparent about that.